release-tools/scripts/setup_runners_service_windows.ps1

```diff
@@ -22,14 +22,16 @@
 
 # setup user parameters
 $SETUP_ACTION=$args[0]
-if (!$SETUP_ACTION) {
-  echo "Please enter 1 parameter: --es | --es-nosec | --kibana | --kibana-nosec"
+$OD_VERSION=$args[1]
+if (!$SETUP_ACTION -or !$OD_VERSION) {
+  echo "Please enter 2 parameters: --es | --es-nosec | --kibana | --kibana-nosec 1.12.0"
```

Comment:
Why are we hardcoding the version --kibana-nosec 1.12.0

Reply:
That is the help message as an example, but can be changed to things like <ODFE Version> for more clarity.

Reply:
Can we have it generic like 1.x.x instead of a specific version.

Reply:
updated